src/measure_theory/constructions/borel_space.lean

```diff
@@ -170,6 +170,43 @@ the `σ`-algebra of measurable sets is exactly the `σ`-algebra generated by ope
 class borel_space (α : Type*) [topological_space α] [measurable_space α] : Prop :=
 (measurable_eq : ‹measurable_space α› = borel α)
 
+namespace tactic.interactive
+
+open tactic
+setup_tactic_parser
+
+/--
+Turn a topological space into a measurable space with the `borel` `σ`-algebra.
+(Almost) equivalent to `letI := borel α, haveI : borel_space α := ⟨rfl⟩`.
+
+If `α` already has a `measurable_space` instance which is equal (but possibly not definitionally
+equal) to `borel α`, then `borel α` turns this equality into defeq by code (almost) equivalent to
+`have h : ‹measurable_space α› = borel α := borel_space.measurable_eq, subst h, letI := borel α`.
+-/
+meta def borel (t : parse texpr) : tactic unit :=
```

Comment:
I am not able to check the tactic code here, but I am willing to trust you on this one. I'd like to have a different name, though, because `borel E` is already the name of the sigma-algebra. What about `borelize E`?

Reply:
Done. I'll try to make `borelize [X, Y]` work tomorrow.